### ISSUE TITLE
Wrong formatted SD cards can be the cause why it's not working

### ISSUE THREAD

**Issue by codebucketdev** [codebucketdev/magisk-foldermount#7]
Ok folks, some people still reported issues with not properly mounted folders and I asked them to send me a screenshot with the output of `mount`.

So the reason why it's not working is because their SD cards are formatted using Windows proprietary exFAT, which is known for causing problems on Linux kernel based systems like Android does. In that case I would suggest you to reformat the SD card using FAT32 (vfat) on a PC and then it should work.

**TL;DR:** SD cards formatted using exFAT aren't compatible and therefore not supported by me.
